accounts/views.py

```diff
@@ -11,7 +15,23 @@ class RegistrationView(CreateView):
     template_name = 'registration/register.html'
 
     def form_valid(self, form):
+        form = self.form_class(data=self.request.POST)
```

Comment:
Why are you recreating the form instance, when a validated one is already passed to this method?